doc/help/elektra-data-structures.md

```diff
@@ -366,3 +366,80 @@ The algorithm uses the `syncbits` to decide if the key set needs to be
 synchronised.
 
 Continue reading [with the error handling](elektra-error-handling.md).
+
+## Order Preserving Minimal Perfect Hash Map
+
+### Vstack
+
+The `Vstack` is a virtual stack implementation with dynamical memory allocation.
+The allocation doubles the space if the `Vstack` is full and reduces the space by the half
+if the memory usage gets under a quarter.
+
+The data will be stored as void pointer and can be inserted with the push call.
```

Comment:
A small tip: If you use the word invocation instead of call it cannot be confused with the verb call.
(Not necessary to be changed here: it is clear as it is)